reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -1,10 +1,28 @@
 package com.innowisegroup.reelpicker.datetime
 
+import com.innowisegroup.reelpicker.extension.isWithinMinMaxRange
 import com.innowisegroup.reelpicker.extension.requireNonNull
 import java.io.Serializable
 import java.util.*
 
-class LocalTime(val hour: Int, val minute: Int) : Serializable {
+class LocalTime(hour: Int, minute: Int) : Serializable {
```

Comment:
make the class constructor private and remove internal fields and init block^

class LocalTime private constructor(val hour: Int, val minute: Int) : Serializable {....}